integrated_channels/integrated_channel/views.py

```diff
@@ -0,0 +1,50 @@
+"""
+Views for Enterprise Integrated Channel.
+"""
+
+from __future__ import absolute_import, unicode_literals
+from rest_framework.views import APIView
+
+
+class PushLearnerDataToIntegratedChannel(APIView):
+    """
+    Provide an endpoint to route learner data to the appropriate channels based on Enterprise's configuration.
+    """
+
+    def post(self, request):
+        """
+        Hit when certificates are generated or when audit learners in self paced courses reach a passing grade.
+
+        When this happens, this method will route the learners data appropriately.
+
+        For a given learner who has completed a course, find out the following:
+            if the learner is associated with an enterprise customer.
+            if the course is part of an enterprise enrollment.
+            if the enterprise customer is configured for an Integrated Channel.
```

Comment:
...but if the learner isn't associated with an Enterprise Customer then I guess that trumps the EC check